plantdescriptionengine/example-plant-description.json

```diff
@@ -0,0 +1,87 @@
+{
+    "include": [],
+    "plantDescription": "ArrowHead core",
+    "active": true,
+    "systems": [
+        {
+            "systemName": "gatekeeper",
+            "systemId": "gatekeeper"
+        },
+        {
+            "systemName": "pde",
+            "systemId": "pde",
+            "ports": [
+                {
+                    "portName": "monitorable",
+                    "serviceDefinition": "monitorable",
+                    "consumer": true
+                }
+            ]
+        },
+        {
+            "systemName": "test_system",
```

Comment:
If this systemName appears as system name in ServiceRegistry, please note that from version 4.4.0 underscore in system names is not valid, Service Registry will reject requests with such system names (see issue #202).

Reply:
Fixed